stock_prices/stock_prices.py

```diff
@@ -1,15 +1,35 @@
 #!/usr/bin/python
 
 import argparse
+import sys
+
 
 def find_max_profit(prices):
-  pass
+    max_profit = float("-inf")
+    for index, selling_price in enumerate(prices):
```

Comment:
Oh I forgot that enumerate also gives you an automatic counter. But it seems to me that looping over prices twice just to get the index is pretty inefficient. Once you finish up the other exercises try and come back to this one and see if you can optimize it. From what I can see you won't have to change much other than add a min price but feel free to try other strategies.